Wiki reorg: Signer Page (#4600)
w3f/polkadot-wiki: docs/general/polkadot-vault.md

---
id: polkadot-vault

> title: Parity Signer (aka Polkadot Vault)

Reviewer: ```suggestion
title: Polkadot Vault (formerly Parity Signer)
```

sidebar_label: Polkadot Vault
description: Learn about the Polkadot Vault
keywords: [parity signer, signer, polkadot vault]
slug: ../polkadot-vault
---

:::info

> The Polkadot Vault app is a cold storage solution for power users. For other users, we recommend

Reviewer: Isn't Polkadot Vault intended for a common user? I think we need not limit this to just power users.

```suggestion
The Polkadot Vault app is an air-gapped cold storage solution for power users. For other users, we recommend
```

using [Ledger](./ledger.md) devices.

:::

> [Parity Signer](https://www.parity.io/technologies/signer) (aka Polkadot Vault) is a cold storage

Reviewer: Super strange that it is still referred to as Parity Signer on the website and Android!

```suggestion
[Polkadot Vault](https://www.parity.io/technologies/signer) (aka Parity Signer) is a cold storage
```

solution that allows you to use a phone in airplane mode as an air-gapped wallet. The Vault App is
not technically a wallet as it does not allow to transfer funds. It is more of a key-chain tool that
will enable you the create, manage, and restore accounts.

By default, the Vault app contains chain specifications for Polkadot, Kusama, and Westend. Metadata

> updates are possible via a QR code fountain. It is also possible to add chains and do metadata

Reviewer: Add markdown link to the  updates using QR code fountain section in the other doc.

```suggestion
updates are possible via a QR code fountain. It is also possible to add other Substrate based chains and do metadata
```

| Ledger app updates on Ledger Live sometimes lag behind chain updates resulting in users only being able to transact if they [install developer versions](./ledger.md#install-the-developer-release) (only for advanced users). |                                                                                                         Metadata updates are always available to install as soon as they are released, but the installation process requires familiarity with Linux commands.                                                                                                         |

## Create and Import Accounts

> 

Reviewer: The recommendation should also be to use the Parity Signer companion https://chrome.google.com/webstore/detail/parity-signer-companion/damllfnhhcbmclmjilomenbhkappdjgb

Author: right! totally forgot about it

---
id: polkadot-vault
title: Polkadot Vault (formerly Parity Signer)
sidebar_label: Polkadot Vault
description: Learn about the Polkadot Vault
keywords: [parity signer, signer, polkadot vault]
slug: ../polkadot-vault
---

:::info

> The Polkadot Vault app is an air-gapped cold storage solution for all users including developers and

Reviewer 2: We should mention that Polkadot Vault is currently available only on iOS

|              Hardware designed to stay offline and be secure. Certified by French cybersecurity agency [ANSSI](https://www.ssi.gouv.fr/en/cybersecurity-in-france/the-national-cybersecurity-agency-of-france/).               |                                                                                                                 Hardware not designed to stay offline. The user must switch off all inbound and outbound connections (network, wifi, Bluetooth, NFC).                                                                                                                 |
|                                 Hardware is resistant to side-channel attacks via [Secure Element](https://www.ledger.com/academy/security/the-secure-element-whistanding-security-attacks/).                                  |                                                                                                                                                If there is no wired connection, side-channel attacks are not possible.                                                                                                                                                |

> |                                                                              One mnemonic seed phrase for all intra- and inter-currency accounts.                                                                              | One mnemonic for each account or one mnemonic for multiple accounts via [account derivation](../learn/learn-account-advanced.md#derivation-paths) or default [Substrate address format](../learn/learn-account-advanced.md#address-format) (not recommended to use the same account on different chains). For `n` chains, a minimum of `n` mnemonics are recommended. |

Reviewer 2: > (not recommended to use the same account on different chains). For `n` chains, a minimum of `n` mnemonics are recommended.

One could argue that this is one of the great features of Substrate and it _is_ recommended. But really there's no recommendation here; it depends on the user's needs. Check [this article](https://support.polkadot.network/support/solutions/articles/65000103707-can-i-use-the-same-account-on-polkadot-kusama-and-parachains-#Should-I-use-the-same-account-across-different-chains?).

> |                                                                          Easy firmware and application upgrades through the Ledger Live application.                                                                           |                                                                                    For [app upgrades](#update-the-vault-app), users must factory reset the phone and recover all accounts through seed phrases. Metadata updates for each chain must be done via QR code fountain.                                                                                    |

Reviewer 2: It would be nice to mention that you don't _have to_ update the app. You can use the same version for ever; it won't stop working if you don't update.

> |                                                                                          Currently, not all parachains are supported.                                                                                          |                                                                                                               Users can add all parachains if they have the wss endpoint and know [how to extract the chain specifications and metadata](#add-chains).                                                                                                                |

Reviewer 2: If the chain is provided by a third-party they can add it from there, without having to do any power user stuff. The only reasons a user might want to do it themselves are:
1. Trust
2. The chain isn't provided by any third party (example Statemin*)

So, I would rephrase this to sth like:
"Users can add all parachains either through a third-party provider, or if they have the wss endpoint and know [how to extract the chain specifications and metadata](#add-chains)."

> | Ledger app updates on Ledger Live sometimes lag behind chain updates resulting in users only being able to transact if they [install developer versions](./ledger.md#install-the-developer-release) (only for advanced users). |                                                                                                         Metadata updates are always available to install as soon as they are released, but the installation process requires familiarity with Linux commands.                                                                                                         |

Reviewer 2: Same here. As long as the metadata is provided by a third party (and the user has added the chain from them), they don't need to go through the manual process. Additionally to the above, one reason the user might want to do it themselves is that the provider doesn't update the metadata quickly (pretty much like Ledger).

So, I would rephrase it to sth like: 
"Metadata updates are always available to install as soon as they are released, either through the third-party provider, or manually, in which case the installation process requires familiarity with Linux commands."

## Create and Import Accounts

We recommend to import a Vault account into the
[Polkadot-JS Browser Extension](./polkadotjs.md#polkadot-js-extension) or

> [Parity Signed Companion](https://chrome.google.com/webstore/detail/parity-signer-companion/damllfnhhcbmclmjilomenbhkappdjgb).

Reviewer 2: Parity Signe**r** Companion (although honestly I don't find the point of this extension)

Alternatively you can import a Vault account directly into the Polkadot-JS UI but keep in mind that
clearing the cache of your browser will remove those accounts (this will not happen for accounts
imported into an extension). See the instructions in

> [this video tutorial](https://youtu.be/hgv1R9mPEXw) to learn more about creating and importing Vault

Reviewer 2: "See the instructions in
 [this articlel](https://support.polkadot.network/support/solutions/articles/65000184076-polkadot-vault-how-to-create-an-account) to learn more about creating Vault accounts, and [this article](https://support.polkadot.network/support/solutions/articles/65000184118-polkadot-vault-how-to-add-your-account-on-polkadot-js-ui) on how to import them to the Polkadot extension or Polkadot-JS UI."
 
 The process for Vault is a bit different from Signer. Having a video for it would be nice.

accounts.

## Signing Extrinsics

Remember to always check for metadata updates before signing transactions. See

> [this video tutorial](https://youtu.be/gbvrHzr4EDY) to learn how to transfer funds and do metadata

Reviewer 2: "See [this article](https://github.com/w3f/polkadot-wiki/pull/4600/files#diff-5d4d0a286cdc7b1d016ee155f9694dbcddc13f5264490fc1a960c38000baca4d) to learn how to sign transactions and [this article](https://support.polkadot.network/support/solutions/articles/65000184128-polkadot-vault-how-to-add-a-new-chain-and-update-the-metadata) on how to easily add new chains and do metadata updates using the Vault App."

updates using the Vault App.

The procedure to sign transactions with the Vault App is as follows:

- The wallet or browser extension will show a QR code encoding the information about what you are
  going to sign.
- After scanning the QR code with the Vault App, you will be presented with decoded information

>   about what you will sign. Make sure the information matches what you asked to sign in the first

Reviewer 2: "Make sure the information matches what you **intended** to sign in the first..."

- recover your accounts (Add Key Set > Recover Key Set).

## Security Notes

The Vault App has a Log that will tell you all activities performed with it. It is important to mark
down the last action you did so that you can do a security check the next time you use the app.
Also, the Vault App will always tell you if the phone has been (even briefly) connected to the
internet. In case of an unrecognized connection it is recommended to:

- Backup your accounts (i.e. make sure you have the mnemonic seed phrases)
- Follow the steps in [Update the Vault App](#update-the-vault-app)
- Once offline, create a new account on the Vault App
- Import the compromised accounts and transfer the funds to the new non-compromised account.

If an account's private key has been exported from the Vault App, the public key will be marked as
"hot" and the following message will be displayed _This key is marked hot because its root private
key has been exported_.

## Add Chains

The Vault App contains chain specs for Polkadot, Kusama, and Westend by default. It is possible to
add more chains via QR-code, and update their metadata by generating your own QR-code fountain in a

> metadata portal similar to that [signed by Parity](https://metadata.parity.io/#/polkadot).

Reviewer 2: Add to that: "Alternatively you can use a third-party provider to add chains and their metadata. Check [this article](https://support.polkadot.network/support/solutions/articles/65000184128-polkadot-vault-how-to-add-a-new-chain-and-update-the-metadata) for detailed instructions. If you choose this approach, you should trust the provider you choose."

:::info Advanced How-to Guides

See the [advanced guides](../learn/learn-vault-guides.md) for more information about how to add

> chain spec and do metadata updates.

Reviewer 2: "chain spec and do metadata updates yourself."

---
id: polkadot-vault
title: Polkadot Vault (formerly Parity Signer)
sidebar_label: Polkadot Vault
description: Learn about the Polkadot Vault
keywords: [parity signer, signer, polkadot vault]
slug: ../polkadot-vault
---

> :::info
> 
> The Polkadot Vault app is currently available only for iOS.

Reviewer: Kind of misleading. It is live on Android as well, but only rebranded on iOS

```suggestion
:::info Rebranding of Parity Signer to Polkadot Vault

The Polkadot Vault app rebranding is currently live only on iOS.
```

|                                 Hardware is resistant to side-channel attacks via [Secure Element](https://www.ledger.com/academy/security/the-secure-element-whistanding-security-attacks/).                                  |                                                                                                                                                                                                                                                                            If there is no wired connection, side-channel attacks are not possible.                                                                                                                                                                                                                                                                             |

> |                                                                              One mnemonic seed phrase for all intra- and inter-currency accounts.                                                                              | One mnemonic for each account or one mnemonic for multiple accounts via [account derivation](../learn/learn-account-advanced.md#derivation-paths) or default [Substrate address format](../learn/learn-account-advanced.md#address-format). Although the general recommendation is to use different accounts on different chains, the final choice depends on the user's needs. See [this support article](https://support.polkadot.network/support/solutions/articles/65000103707-can-i-use-the-same-account-on-polkadot-kusama-and-parachains-#Should-I-use-the-same-account-across-different-chains?) for more information. |

Reviewer: Avoid the usage of terms like "currency".  Also, Ledger derives all the accounts through a single seed phrase right? If not correct the update I made. Also, pointing the user to the support article without making a general recommendation.

```suggestion
|                                                                              Accounts derived from one mnemonic seed phrase.                                                                              | One mnemonic for each account or one mnemonic for multiple accounts via [account derivation](../learn/learn-account-advanced.md#derivation-paths) or default [Substrate address format](../learn/learn-account-advanced.md#address-format). See [this support article](https://support.polkadot.network/support/solutions/articles/65000103707-can-i-use-the-same-account-on-polkadot-kusama-and-parachains-#Should-I-use-the-same-account-across-different-chains?) for information on whether to use the same account or different accounts on different chains based on your needs. |
```

> |                                                                          Easy firmware and application upgrades through the Ledger Live application.                                                                           |                                                                                                                                                    No need to update the app as the same version will work fine forever and will not stop working if you do not update it. For [app upgrades](#update-the-vault-app), users must factory reset the phone and recover all accounts through seed phrases. Metadata updates for each chain must be done via QR code fountain.                                                                                                                                                     |

Reviewer: Removing the "works forever" part and mentioning why the users may need a factory reset.

```suggestion
|                                                                          Easy firmware and application upgrades through the Ledger Live application.                                                                           |                                                                                                                                                    The app should never be connected to the internet after installation, so the version installed on the phone should not be updated directly. For [app upgrades](#update-the-vault-app), users must factory reset the phone and recover all accounts through seed phrases. Metadata updates for each chain must be done via the QR code fountain.                                                                                                                                                     |
```

|                                                                                          Currently, not all parachains are supported.                                                                                          |                                                                                                                                                                                                                       Users can add all parachains either through a third-party provider or if they have the wss endpoint and know [how to extract the chain specifications and metadata](#add-chains).                                                                                                                                                                                                                        |

> | Ledger app updates on Ledger Live sometimes lag behind chain updates resulting in users only being able to transact if they [install developer versions](./ledger.md#install-the-developer-release) (only for advanced users). |                                                                                                                                                                                                      Metadata updates are always available to install as soon as they are released, either through the third-party provider or manually, in which case the installation process requires familiarity with Linux commands.                                                                                                                                                                                                      |

Reviewer: ```suggestion
| Ledger app updates on Ledger Live sometimes lag behind chain updates resulting in users only being able to transact if they [install developer versions](./ledger.md#install-the-developer-release) (only for advanced users). |                                                                                                                                                                                                      Metadata updates are always available to install once they are released, either through the third-party provider or manually. In this case, the installation process requires familiarity with working on the command prompt.                                                                                                                                                                                                      |
```
